src/recipe/dto/create-recipe-request.ts

```diff
@@ -21,4 +21,10 @@ export class CreateRecipeRequest {
     example: 'Step 1\nStep 2\nStep 3',
   })
   preparation: string;
+
+  @ApiProperty({
+    description: 'Sets recipe to public',
```

Comment:
maybe something like this?
```suggestion
    description: 'Sets recipe to be publicly accessible by other users',
```